app/frontend/wallet/shelley/shelley-bitbox02-crypto-provider.ts

```diff
@@ -313,16 +244,17 @@ const ShelleyBitBox02CryptoProvider = async ({
       })
     }
 
-    const response = await withDevice(async (bitbox02: BitBox02API) => {
-      return await bitbox02.cardanoSignTransaction({
-        network: bb02Network,
+    const response = await withDevice(async (pairedBitbox) => {
+      return await pairedBitbox.cardanoSignTransaction({
+        network: selectedNetwork,
         inputs,
         outputs,
-        fee: txAux.fee.toString(),
-        ttl: txAux.ttl?.toString() ?? null,
+        fee: txAux.fee as unknown as bigint,
```

Comment:
We have these `as unknown as bigint` assertions (around 6 in this file). 

The reason is, the values (in this case it's `txAux.fee`) are of type "BigNumber" which [comes from a package called bignumber.js](https://mikemcl.github.io/bignumber.js/) and we need to somehow assert its type to be `bigint` to work with the current `bitbox-api`.

If `bitbox-api` allows `bigint | string` for these, then we can remove the assertions.